tests/drivers/build_all/virtio/prj.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_TEST=y
+CONFIG_VIRTIO=y
+CONFIG_PCIE=y
+CONFIG_VIRTIO_PCI=y
```

Comment:
defaults to `y` when DT node has `status = "okay"`
```suggestion
```

Reply:
Done